src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -103,6 +103,9 @@ internal static bool IsPrimitiveType(this CorElementType et)
         // The following intrinsics return true if input is a compile-time constant
         // Feel free to add more overloads on demand
 #pragma warning disable IDE0060
+        [Intrinsic]
+        internal static bool IsKnownConstant(Type? t) => false;
+
```

Comment:
```suggestion
```
This does not seem to be used anywhere.

Reply:
I've also wanted to reintroduce the GetTypeCode optimization using this in another commit.